transport.go

```diff
@@ -104,6 +104,10 @@ type Transport struct {
 	// Tracer.Close is called when the transport is closed.
 	Tracer *logging.Tracer
 
+	// IsNonQuicPacket is a custom function that can be set to override the detection of
+	// non-QUIC packets. If not set, default DefaultIsNonQuicPacket handler is used
+	IsNonQuicPacket func(packet []byte) bool
```

Comment:
```suggestion
	IsNonQUICPacket func(packet []byte) bool
```

Same for `DefaultIsNonQUICPacket` below.